src/ccid_usb.c

```diff
@@ -1622,6 +1617,10 @@ void InterruptStop(int reader_index)
 		if (ret < 0)
 			DEBUG_CRITICAL2("libusb_cancel_transfer failed: %s",
 				libusb_error_name(ret));
+	} else {
+		// Indicate that the next attempt to start an interrupt transfer shouldn't
```

Comment:
The potential problem is whether it's possible that `InterruptStop()` is called after the corresponding `InterruptRead()` already exited; in that case we'll incorrectly abort the next polling transfer later.

If that's possible indeed, I'm not 100% sure what should be the proper fix.